examples/uros_6lowpan/uros_6lowpan_main.c

```diff
@@ -0,0 +1,197 @@
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+//uROS libraries
+#include <rcl/rcl.h>
+#include <rcl/error_handling.h>
+#include <std_msgs/msg/int32.h>
+#include <rmw_uros/options.h>
+
+#if defined(BUILD_MODULE)
+int main(int argc, char *argv[])
+#else
+int uros_6lowpan_main(int argc, char* argv[])
+#endif
+{
+    char buffer[256]; // Buffer to save configuration commands.
+    char aux_buffer[2]; // Buffer to save the configuration ID.
+
+    if(3 > argc || 0 == atoi(argv[2]))
+    {
+        printf("usage: program [-h | --help] | ip port sub/pub [<max_topics>]\n");
+        return 0;
+    }
+    //Check if the IP and the port are correct
+    if(strlen(argv[1])>39 || strlen(argv[2])>4){
+        printf("Error: IP or port size incorrect \r\n");
+        return 0;
+    }
+
+    //6lowpan configuration process
+    printf("Do you want to configure the 6lowpan network? (Y/N)\r\n");
+    memset(buffer,0,sizeof(buffer));
+    scanf("%s", buffer);
+
+    if(!strcmp(buffer,"y")){
+        system("ifdown wpan0"); // Is necessary to bring down the network to configure.
+        system("i8sak wpan0 startpan cd:ab"); //Set the radio as an endpoint.
+        system("i8sak set chan 26"); //Set the radio channel.
+        system("i8sak set panid cd:ab"); //Set network PAN ID.
+
+        // Set the ID
+        printf("Introduce your 6LowPan ID (It must between 00 and FF (Hex))\r\n");
+        scanf("%s", aux_buffer);
+        if (strlen(aux_buffer) < 2) {
+            printf("Error ID must be between 00 and FF\n");
```

Comment:
```suggestion
            printf("Error ID must have length 2 and be between 00 and FF\n");
```

btw, this does not really verify that the content is a valid hexadecimal number...